### DISCUSSION TITLE
This should be in the FAQ section: How to update credential cache on Windows after changing domain password

### DISCUSSION THREAD

**Discussion by SetTrend** [git-for-windows/git#4043] [Q&A]
I couldn't find any help fitting this issue on the web:

After I updated my domain password, `git` isn't able to connect anymore. I get an `fatal: Authentication failed for 'https://azuredevops.…'` error message.

Folks on the web suggest to remove my credentials from Windows Credential Manager, but on my machine there are no credentials stored in Windows Credential Manager:

![Windows Credential Manager](https://user-images.githubusercontent.com/9283914/192777618-3c82011c-d56d-463d-99e8-6418861592ba.png)

It's blank.

I request to suggest remedy and to add your proposed solution to the FAQ.

**Comment by landstander668** [comment]
I'm thinking you might have a different credential helper enabled. What's the output from `git config --show-origin --get-regexp '^credential\.'`?

**Reply by SetTrend** [reply]
Thank you for trying to help!

Here's the output:

```powershell
PS D:\Documents\***> git config --show-origin --get-regexp '^credential\.'
file:C:/Program Files/Git/etc/gitconfig credential.helper manager-core
file:C:/Program Files/Git/etc/gitconfig credential.https://dev.azure.com.usehttppath true
file:C:/Users/Axel/.gitconfig   credential.https://azuredevops.***.de.provider generic
```

(I cloaked my customer's Id.)

I'm trying to reach the 3<sup>rd</sup> provider (`https://azuredevops.***.de`).

**Reply by SetTrend** [reply]
Apparently it takes Visual Studio 2022 to update the credentials accordingly. When I opened a Visual Studio solution, Visual Studio 2022 asked me for my Azure DevOps credentials. After I entered my credentials there, everything went fine.

Still, there should be some documentation on how to achieve this without using Visual Studio 2022.
